addons/mail/static/src/components/discuss/tests/discuss_tests.js

```diff
@@ -2882,185 +2870,6 @@ QUnit.test('post a simple message', async function (assert) {
     );
 });
 
-QUnit.test('post message on non-mailing channel with "Enter" keyboard shortcut', async function (assert) {
```

Comment:
This test must stay, as it concerns non-mailing channel. The fact it is a non-mailing channel can be removed from the title though.